feature-libs/order/components/order-details/order-overview/order-overview.component.spec.ts

```diff
@@ -486,4 +493,48 @@ describe('OrderOverviewComponent', () => {
       expect(address).toEqual(mockFormattedAddress);
     });
   });
+
+  describe('when unit order is defined', () => {
+    beforeEach(() => {
+      component.order = mockOrder;
+      spyOn(translationService, 'translate').and.returnValue(of('test'));
+    });
+
+    it('should call getBuyerNameCardContent(customer: B2BUser)', () => {
+      spyOn(component, 'getBuyerNameCardContent').and.callThrough();
+
+      component
+        .getBuyerNameCardContent(mockOrder.orgCustomer)
+        .subscribe((data) => {
+          expect(data).toBeTruthy();
+          expect(data.title).toEqual('test');
+          expect(data.text).toEqual([
+            mockOrder.orgCustomer?.name,
+            '(' + mockOrder.orgCustomer?.uid + ')',
+          ]);
+        })
+        .unsubscribe();
+
+      expect(component.getBuyerNameCardContent).toHaveBeenCalledWith(
+        mockOrder.orgCustomer
+      );
+    });
+
+    it('should call getUnitNameCardContent(orgUnit: string)', () => {
+      spyOn(component, 'getUnitNameCardContent').and.callThrough();
+
+      component
+        .getUnitNameCardContent(mockOrder.orgUnit.name)
+        .subscribe((data) => {
+          expect(data).toBeTruthy();
+          expect(data.title).toEqual('test');
+          expect(data.text).toEqual([mockOrder.orgUnit.name]);
+        })
+        .unsubscribe();
+
+      expect(component.getUnitNameCardContent).toHaveBeenCalledWith(
+        mockOrder.orgUnit.name
+      );
+    });
+  });
```

Comment:
i don't see any test that use new ngOnInit method for order-overview component. Test code coverage is missing there

Reply:
updated, I added a test for ngOnInit, now it is covered